shell/main.js

```diff
@@ -25,6 +25,9 @@ const mb = menubar({
     minHeight: 600,
     backgroundColor: '#FAFAFA',
     webPreferences: {
+      // TODO needs enabling ?
+      // nodeIntegration: false,
+      // contextIsolation: true
```

Comment:
Do we need those?

Reply:
It was more of a reminder to my self to ask about it. The electron docs suggest that those 2 properties be set for security reasons but there is a lot of disagreement about it when reading forums / blogs. I don't quite understand the drawbacks / benefits, do you have any thoughts?

Reply:
I can try them in isolation after we merge and go through beta.